packages/core/src/statistics/Statistics.ts

```diff
@@ -2,26 +2,32 @@ import { Logger } from '@stryker-mutator/api/logging';
 import { commonTokens, tokens } from '@stryker-mutator/api/plugin';
 import { HttpClient } from 'typed-rest-client/HttpClient';
 
-const AZURE_URL = 'https://localstrykertryout.azurewebsites.net/api/HttpTrigger?code=4GWK/KLC6JRlec96851wMgMsD2JkVePiaALWw6lKv4R3RZiKf0xp0w==';
+import { JsonLoader } from './JsonLoader';
+
+const AZURE_URL = 'https://strykerstatistics.azurewebsites.net/api/ReceiveStatistics?code=jVZfGmoB6ofRPa/yPdN/mAOCd6ia67XQkTmLaGWCzlxO5a32PlLj6A==';
 
 export class Statistics {
-  public static inject = tokens(commonTokens.logger, 'httpClient');
-  public statistics: any = {};
+  public static inject = tokens(commonTokens.logger, 'httpClient', 'testRunner');
+  public statistics: Record<string, any> = {};
```

Comment:
Dit is niet heel type-safe. Het is erg makkelijk om typfouten te maken bij b.v. `addStatistic('scorre', ...`. Kan je een interface definieren voor hoe de statistieken er uiteindelijk uit gaan zien?